Source/r_main.c

```diff
@@ -224,6 +224,9 @@ angle_t R_PointToAngleCrispy(fixed_t x, fixed_t y)
     0;
 }
 
+// WiggleFix: move R_ScaleFromGlobalAngle to r_segs.c,
+// above R_StoreWallRange
+#if 0
```

Comment:
Please remove this code, instead of commenting it out, but leave the comment in place.